adserver/templates/adserver/publisher/payouts.html

```diff
@@ -0,0 +1,52 @@
+{% extends "adserver/base.html" %}
```

Comment:
Maybe call this template `payout_list.html`? I got confused with just the plural.